cmd/vela-artifactory/docker_promote.go

```diff
@@ -72,14 +77,25 @@ func (p *DockerPromote) Exec(c *Config) error {
 	}
 
 	for _, payload := range payloads {
-		logrus.Debugf("Promoting target tag %s", *payload.TargetTag)
+		logrus.Debugf("Promoting target tag %s", payload.GetTargetTag())
 
 		_, _, err := client.Docker.PromoteImage(p.TargetRepo, payload)
 		if err != nil {
 			return err
 		}
+		if p.PromotedImageFlag {
```

Comment:
Reviewdog is complaining here because of a missing space.

```suggestion
		}
		
		if p.PromotedImageFlag {
```